### DISCUSSION TITLE
DISCUSSION: DataFusion Meetup in Australia

### DISCUSSION THREAD

**Discussion by alamb** [apache/datafusion#10349] [Ideas]
After our first meetup, (see https://github.com/apache/datafusion/discussions/8152 / https://github.com/apache/datafusion/discussions/8522) I have heard there may be interest in other meetups in different geographies. 

I would personally love to support such meetups (will promote, try to attend, can help find sponsors, etc) as much as possible but I don't have the bandwidth to actually organize them.

Anyone who is interested in attending or helping to organize please leave a comment

Other meetup discussions:
https://github.com/apache/datafusion/discussions/10347

**Comment by lukekim** [comment]
Spice AI could help organize this in Sydney if we get enough interest.
